audius-cli

```diff
@@ -272,9 +272,10 @@ def health_check(ctx, service):
 @click.option("-y", "--yes", is_flag=True)
 @click.option("--seed", is_flag=True)
 @click.option("--chain", is_flag=True)
+@click.option("-b", "--block-explorer", is_flag=True)
```

Comment:
imo we should use the plugin system for this

Reply:
https://github.com/AudiusProject/audius-docker-compose/blob/stage/discovery-provider/docker-compose.plugins.yml#L45

e.g. these containers will only start up if you have run
```
audius-cli register-plugin block-explorer
```

Reply:
good call, will add